README.adoc

```diff
@@ -77,7 +77,7 @@ button to proceed. Use the `confAdmin` and `microprofile` credentials to log in.
 the standard metrics and the application metrics in a text format.
 
 To see only the application metrics, point your browser to `https://localhost:9443/metrics/application`.
-Because these metrics are application-related, you had to point first to the `\http://localhost:9080/inventory/systems` URL,
+Because these metrics are application-related, you have to point to the `\http://localhost:9080/inventory/systems` URL,
```

Comment:
Ok, this is good, nice and clean.

Reply:
Fixed with what is suggested in the issue